### DISCUSSION TITLE
can talos do cache on disk?

### DISCUSSION THREAD

**Discussion by Nyuuk** [siderolabs/talos#9195] [Q&A] [answered]
Can Talos cache on disk instead of memory? Because each node I provide has limited RAM. I installed Talos on Proxmox, and I found that if Talos is not rebooted periodically, the cache will grow, and I think this will have an impact. So, how can I make Talos cache on disk?
![image](https://github.com/user-attachments/assets/708c73a3-646e-4d9a-b20f-535e92f21b2f)

**Comment by smira** [comment, marked as answer]
This is the way Linux manages and reports memory, and it has nothing to do with Talos. You want the OS to use the RAM in an efficient way. Cached memory is not _used_ memory.

See e.g. https://unix.stackexchange.com/questions/263764/what-is-difference-between-cached-memory-and-used-memory or use Google to find more in-depth explanation of various memory usage reporting.
